config/webpack/paths.yml

```diff
@@ -0,0 +1,31 @@
+# Restart webpack-watcher or webpack-dev-server if you make changes here
+default: &default
+  config: config/webpack
```

Comment:
I think we should consider sticking with the ReactOnRails convention here and just keeping all node-related stuff, including the webpack configuration files, in the client folder. That way it will be much more familiar-looking?